### PR TITLE
Add weights to losses

### PR DESCRIPTION
**Pull request by devisperessutti**
NB: [PR12](https://github.com/sentinel-hub/eo-flow/pull/12) should be merged first

PR that allows users to specify weights for each class that will be multiplied with the loss function. 

This is useful to prevent `NO_DATA` labels to drive the gradient descent

### CODE REVIEW

File: eoflow/models/losses.py [hunk 1]
```diff
@@ -2,6 +2,57 @@
...
+
+
+class CategoricalCrossEntropy(Loss):
+    """ Wrapper class for cross-entropy with class weights """
+    def __init__(self, from_logits=True, class_weights=None, reduction=Reduction.AUTO, name='FocalLoss'):
+        """Categorical cross-entropy.
+
+        :param from_logits: Whether predictions are logits or softmax, defaults to True
+        :type from_logits: bool
+        :param reduction: reduction to be used, defaults to Reduction.AUTO
```
Comment (lojzezust) [inline eoflow/models/losses.py]: Missing docstring for `class_weights`.

File: eoflow/models/losses.py [hunk 2]
```diff
@@ -10,7 +61,8 @@ class CategoricalFocalLoss(Loss):
         Keras implementation: https://github.com/umbertogriffo/focal-loss-keras
     """
 
-    def __init__(self, gamma=2., alpha=.25, from_logits=True, reduction=Reduction.AUTO, name='FocalLoss'):
+    def __init__(self, gamma=2., alpha=.25, from_logits=True, class_weights=None, reduction=Reduction.AUTO,
+                 name='FocalLoss'):
```
Comment (lojzezust) [inline eoflow/models/losses.py]: Same as above.

File: eoflow/models/losses.py [hunk 3]
```diff
@@ -46,7 +99,60 @@ def call(self, y_true, y_pred):
...
+    """ Implementation of the Jaccard distance, or Intersection over Union IoU loss.
+
+    Jaccard = (|X & Y|)/ (|X|+ |Y| - |X & Y|)
+            = sum(|A*B|)/(sum(|A|)+sum(|B|)-sum(|A*B|))
+
+    Implementation taken from https://github.com/keras-team/keras-contrib/blob/master/keras_contrib/losses/jaccard.py
+    """
+    def __init__(self, smooth=1, from_logits=True, class_weights=None, reduction=Reduction.AUTO, name='JaccardLoss'):
+        """ Jaccard distance loss.
+
```
Comment (lojzezust) [inline eoflow/models/losses.py]: Here as well.

Comment (devisperessutti) [reply eoflow/models/losses.py]: added

File: eoflow/models/losses.py (hunk 2 above)
Comment (devisperessutti) [reply eoflow/models/losses.py]: added

File: eoflow/models/losses.py (hunk 1 above)
Comment (devisperessutti) [reply eoflow/models/losses.py]: added
